packages/apsara-ui/package.json

```diff
@@ -52,10 +52,18 @@
         "@radix-ui/react-avatar": "^1.0.0",
         "@radix-ui/react-dropdown-menu": "^1.0.0",
         "@tanstack/react-table": "8.5.13",
+        "@rc-component/mini-decimal": "1.0.1",
         "react-query": "3.39.2",
         "react-virtual": "^2.10.4",
         "rc-pagination": "^3.1.17",
         "antd": "^4.16.13",
```

Comment:
So if we remove this, we would need to import lots of packages ?

Does that mean Apsara depend on libraries that are `antd`'s dependencies?

Reply:
removed this dependency as we were using only one type from there.